packages/test-runner-playwright/src/index.ts

```diff
@@ -26,8 +26,20 @@ export function playwrightLauncher(args: PlaywrightLauncherArgs = {}) {
   const {
     product = 'chromium',
     launchOptions = {},
-    createBrowserContext = ({ browser }) => browser.newContext(),
-    createPage = ({ context }) => context.newPage(),
+    createBrowserContext = ({ browser }) => {
+      if ('browserWindow' in browser) {
+        return browser.context();
+      } else {
+        return browser.newContext();
+      }
+    },
+    createPage = ({ context, browser }) => {
+      if ('browserWindow' in browser) {
+        return browser.firstWindow();
```

Comment:
The `firstWindows` is a playwright `Page`. Maybe for Electron we need to open and close windows, instead of making contexts?